airbyte-server/src/main/java/io/airbyte/server/ServerApp.java

```diff
@@ -221,20 +221,11 @@ public static ServerRunnable getServer(final ServerFactory apiFactory, final Con
     final SynchronousSchedulerClient bucketSpecCacheSchedulerClient =
         new BucketSpecCacheSchedulerClient(syncSchedulerClient, configs.getSpecCacheBucket());
     final SpecCachingSynchronousSchedulerClient cachingSchedulerClient = new SpecCachingSynchronousSchedulerClient(bucketSpecCacheSchedulerClient);
-    final SpecFetcher specFetcher = new SpecFetcher(cachingSchedulerClient);
-
-    // todo (cgardens) - this method is deprecated. new migrations are not run using this code path. it
-    // is scheduled to be removed.
-    final Optional<AirbyteVersion> airbyteDatabaseVersion = runFileMigration(
```

Comment:
Nit, we can remove `runFileMigration` as well.

Reply:
According to the description, this PR is trying to just remove the least amount of code possible to stop the migration from being run, so `runFileMigration()` will be removed in a future PR

Reply:
I see, I forgot the description. I checked it in IntelliJ and it is thew only place where it got called, I believe that we can remove this dead code in this review safely. IMO since it is the only place this method is call we should remove it in the same review.

Reply:
Please don't consider it as a blocker.

Reply:
@benmoriceau yeah. it is intentional. in case we've made a mistake and need to bring it back by removing just the call point the chance of having a complex merge conflict on revert is lower, which is why i have taken this approach. if you're okay with that can you switch your review so that gh won't block the PR?

Reply:
I am ok, I just hope that doing that will lead to a lot of dead code If we do it too often. As far as we make sure it is clean up, that's fine.